include/mbedtls/ctr_drbg.h

```diff
@@ -52,7 +52,7 @@
 #define MBEDTLS_CTR_DRBG_KEYSIZE            32 /**< The key size used by the cipher. */
 #else
 #if defined(MBEDTLS_CTR_DRBG_KEY_SIZE_128)
-#warning Warning: using smaller (128bit) key size for CTR DRBG may reduce the security of some operations.
+#pragma message("Warning: using smaller (128bit) key size for CTR DRBG may reduce the security of some operations.")
```

Comment:
Does this message pragma work on all our supported toolchains? Is there a reason we can't use the same mechanism that `MBEDTLS_TEST_NULL_ENTROPY` uses for its warning?